src/broker/broker_config.c

```diff
@@ -350,6 +413,40 @@ broker_config_read_internal (const char *conf_file, T_BROKER_INFO * br_info, int
       return -1;
     }
 
+  /* Validate broker keywords, if keyword is not in list terminate */
+  for (i = 0; i < ini->size; i++)
+    {
+      char *key;
+      bool found = false;
+
+      if (ini->key[i] == NULL)
+        {
+          continue;
+        }
+
+        key = strchr (ini->key[i], ':');
+        if (key == NULL)
+          {
+            continue;
+          }
+
+        key++;
+        for (j = 0; j < broker_keywords_size; j++)
+          {
+            if (strcasecmp (key, broker_keywords[j]) == 0)
+              {
+                found = true;
+              }
```

Comment:
If the key is broker keyword, how about insert "break" ?

Reply:
It seems to be missing while moving the code from ini_parser.c to broker_config.c
Thank you for your comment!